primitives/src/conststr.rs

```diff
@@ -565,13 +565,21 @@ mod ffi {
         }
     }
 
+    // TODO: Shouldn it be cloned?
```

Comment:
It's safe to say that it should be. There's quite a bit of UB we could cause if we didn't do that.